ProcessCIW/ProcessCIW.csproj

```diff
@@ -97,11 +91,15 @@
       <SubType>Designer</SubType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="AppSettings.config" />
+    <None Include="AppSettings.config">
```

Comment:
Copy to output directory was set to do not copy and would not run until I changed it to copy if newer